.github/workflows/main.yml

```diff
@@ -40,6 +40,17 @@ jobs:
           --configuration Release
 
   pack:
+    strategy:
+      matrix:
+        rid:
+          - win-arm64
+          - win-x86
+          - win-x64
+          - linux-x64
+          - osx-arm64
+          - osx-x64
```

Comment:
This is based on the FFmpeg builds we have available (via https://github.com/Tyrrrz/FFmpegBin)